WcaOnRails/spec/models/delegate_report_spec.rb

```diff
@@ -34,6 +34,22 @@
     expect(dr.discussion_url).to eq "https://groups.google.com/a/worldcubeassociation.org/forum/#!topicsearchin/reports/" + URI.encode_www_form_component(dr.competition.name)
   end
 
+  it "wrc_feedback_requested is set false on creation" do
+    dr = FactoryBot.create :delegate_report
+    expect(dr.wrc_feedback_requested).to eq false
+  end
+
+  it "wrc_incidents is required when posted" do
```

Comment:
This says `wrc_incidents is required when posted`, but the report on the test is actually never posted, right? I believe it should say `wrc_incidents is required when wrc_feedback_requested is true` instead.